ambry-tools/src/main/java/com.github.ambry/tools/admin/DirectoryUploader.java

```diff
@@ -146,7 +148,8 @@ public void walkDirectoryToCreateBlobs(String path, FileWriter writer, String da
           throw new IllegalArgumentException("File length is " + f.length() + "; files larger than " + Integer.MAX_VALUE
               + " cannot be put using this tool.");
         }
-        BlobProperties props = new BlobProperties(f.length(), "migration");
+        BlobProperties props =
+            new BlobProperties(f.length(), "migration", Account.UNKNOWN_ACCOUNT_ID, Container.UNKNOWN_CONTAINER_ID);
```

Comment:
We need to fix all tools. AccountId and ContainerId or target account name and target container name should be taken in via config params. But again, as I told earlier, will do it in a  diff PR.